src/OpenTelemetry.Instrumentation.ServiceFabricRemoting/OpenTelemetry.Instrumentation.ServiceFabricRemoting.csproj

```diff
@@ -0,0 +1,34 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
+    <TargetFrameworks>$(NetStandardMinimumSupportedVersion)</TargetFrameworks>
+    <Description>ServiceFabric Remoting instrumentation for OpenTelemetry .NET.</Description>
+    <PackageTags>$(PackageTags);distributed-tracing</PackageTags>
+    <MinVerTagPrefix>Instrumentation.ServiceFabricRemoting-</MinVerTagPrefix>
+  </PropertyGroup>
+
+  <!-- Do not run Package Baseline Validation as this package has never released a stable version.
+  Remove this property once we have released a stable version and add PackageValidationBaselineVersion property. -->
+  <PropertyGroup>
+    <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
+  </PropertyGroup>
+
+  <!-- This is needed because the Service Fabric dlls target AMD64-->
+  <PropertyGroup>
+    <ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>None</ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="$(RepoRoot)\src\Shared\AssemblyVersionExtensions.cs" Link="Includes\AssemblyVersionExtensions.cs" />
+    <Compile Include="$(RepoRoot)\src\Shared\Guard.cs" Link="Includes\Guard.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Options" Version="8.0.2" />
+    <PackageReference Include="Microsoft.ServiceFabric.Actors" Version="7.1.2448" />
+    <PackageReference Include="Microsoft.ServiceFabric.Services.Remoting" Version="7.1.2448" />
+    <PackageReference Include="OpenTelemetry.Api.ProviderBuilderExtensions" Version="$(OpenTelemetryCoreLatestVersion)" />
```

Comment:
@Kielek , our applications are not ready to switch to 1.10 yet, should I change this to 1.9.0 for now?

Also, this code has been running in our production service fabric clusters for three months without any issues. Should I change the stability to RC instead of beta? We would like to start the transition to consuming this nugget as soon as it is ready.

Reply:
It should be fine make 1/couple of release with 1.9.0. Finally, the recommendation is to pin to the macro.

If we speaking about stability. My recommendation for the first release is either `alpha` or (as you heavily tested it) `beta`. 

Going to RC/stable phase should means that all attributes/metrics you produce are semantic version compliant (and sem conv is stable).

Reply:
ok, sounds good. I just changed the version to 1.9.0 for now.